tutorials/navigation/real_time_navigation_3d.rst

```diff
@@ -0,0 +1,64 @@
+.. _doc_real_time_navigation_3d:
+
+Real Time Navigation (3D)
+=========================
+
+Introduction
+------------
+
+Godot provides two nodes which can be used to achieve real time navigation
+in a 3D environment, :ref:`Navigation <class_Navigation>` and :ref:`NavigationMeshInstance <class_NavigationMeshInstance>`.
+
+A navigation mesh instance is used to hold a :ref:`Navigation <class_NavigationMesh>`,
+which defines what ares in a scene can be navigated to. The navigation
```

Comment:
```suggestion
which defines what ares in a scene can be navigated to. The :ref:`Navigation <class_Navigation>`
```
(linking it once more for clarity, and to mark the difference to NavigationMesh in the sentence before)